app/src/main/java/com/kickstarter/viewmodels/ActivityFeedViewModel.java

```diff
@@ -115,15 +114,15 @@ public ViewModel(final @NonNull Environment environment) {
           this.currentUser.isLoggedIn(),
           Pair::create
         )
-        .filter(resumedAndLoggedIn ->
-          resumedAndLoggedIn.first == ActivityEvent.RESUME && resumedAndLoggedIn.second
-        )
+        .map(resumedAndLoggedIn -> resumedAndLoggedIn.second)
+        .filter(loggedIn -> loggedIn)
```

Comment:
I was thinking that I didn't really like it the way it is. I will convert it to use the function, which as you say, is much more declarative and clear.